src/Stratis.Bitcoin.Features.Api/Program.cs

```diff
@@ -16,11 +16,13 @@ public static IWebHost Initialize(IEnumerable<ServiceDescriptor> services, FullN
         {
             Guard.NotNull(fullNode, nameof(fullNode));
 
+            var ApiUri = fullNode.NodeService<ApiSettings>().ApiUri;
```

Comment:
unvar and small `a`piUri

Reply:
Changed to `Uri apiUri = fullNode.NodeService<ApiSettings>().ApiUri;`

Reply:
I think this hasn't been changed?

Reply:
Program.cs was omitted from the commit. Added.